playwright/pages/components/ReviewDialog.ts

```diff
@@ -0,0 +1,49 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+import { Dialog } from "./Dialog";
+
+export class ReviewDialog extends Dialog {
+  readonly messageTextarea: Locator;
+
+  constructor(page: Page) {
+    super(page, {
+      title: "Share feedback",
+    });
+
+    /**
+     * Components
+     */
+    // Handle components here
+
+    /**
+     * Static locators
+     */
+    this.messageTextarea = this.dialog.getByPlaceholder("Tell us more...", {
```

Comment:
Do we have a label to use instead of placeholder? If we do, could we use `getByLabel()` instead of `getByPlaceholder()`?

If not, it's something we may want to raise as an accessibility issue.

Reply:
Yes there is not label and that's not great. It's an old component and when we will redo it we need to change it.